airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcDatabase.kt

```diff
@@ -44,6 +48,7 @@ abstract class JdbcDatabase(protected val sourceOperations: JdbcCompatibleSource
             connection.autoCommit = false
             for (s in queries) {
                 LOGGER.info("executing query within transaction: $s")
+                LOGGER.info(Thread.currentThread().stackTrace.joinToString("\n    "))
```

Comment:
nit: combine this log statement with the previous one, to avoid mixing up concurrent log messages